package.json

```diff
@@ -1,13 +1,17 @@
 {
   "name": "react-social-auth",
-  "version": "1.1.1",
+  "version": "1.1.2",
   "description": "Simple client side social authentication for React applications.",
   "main": "lib/react-social-auth.js",
   "scripts": {
     "build": "webpack -d",
     "test": "babel-node ./node_modules/mocha/bin/_mocha",
     "fmt": "prettier --write *.js; prettier --write '{src,test}/**/*.js'",
-    "lint": "eslint --fix dangerfile.js src/*.js test/*.js"
+    "lint": "eslint --fix dangerfile.js src/*.js test/*.js",
+    "lint:ci": "npm run lint -- -o lint-results.json -f json",
+    "lint:fix": "npm run lint -- --fix",
```

Comment:
Why three separate lint commands?

Reply:
I followed a similar pattern as done in here: https://github.com/smartprocure/futil-js/blob/master/package.json#L16 except that I tried to streamline it. What would you suggest? I don't know when the `lint:fix` command is used. I assume is used by `duti` at some point.